### ISSUE TITLE
[Android] F-Droid release

### ISSUE THREAD

**Issue by joe733** [bluesky-social/social-app#898]
### Is your feature request related to a problem? Please describe.

Yes, can't download android app without Google Playstore.

### Describe the solution you'd like

Bluesky is made available through [F-Droid](https://f-droid.org/).

### Describe alternatives you've considered

[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/info)?

### Additional context

`F-droid release === Peace of mind :)`

**Comment by dmytri** [comment]
update?

**Comment by gagzzz** [comment]
Hello,
I'd be also interested since I have a bluesky account but still cannot use it easily on my phone (no google things here). 

Also I tried with [Obtainium](https://github.com/ImranR98/Obtainium) but it requires published Releases, and bluesky-social/social-app only has one tag.

Please publish releases :)

Thank you!

**Comment by RasheedAZ** [comment]
Are the developers okay with submitting this app for inclusion the F-Droid main repo? 

This is an open source app built on an open protocol... so would be great if it was available on FOSS only app stores like F-Droid.

Thanks :)

**Comment by yozachar** [comment]
> F-droid release === Peace of mind :)

I'm re-thinking about that, after I read https://github.com/GrapheneOS/Camera/issues/227. It's been more than a year, since the last comment was made there. I do not know if the state of things has improved or not.

**Comment by RasheedAZ** [comment]
This really is not the place for it and I wish you had not linked to that thread. Many of us are aware of all those issues, F-Droid's choices and reasons, how apps are packaged in Linux in general, and other personal issues surrounding GOS dev and would still like FOSS apps to be available on FOSS-only platforms and app stores if possible. Besides F-Droid, there are no FOSS-only app stores for Android (other than its forks which are not as well developed). 

For every single one of their criticisms of F-Droid there is a good reply whether they agree with it or not and the accusat

It is good that GOS prioritises security over privacy, decentralisation and avoidance of Big Tech, etc. but different people have different threat models and may prioritise things differently.

**Comment by haileyok** [comment]
@codenyte I responded in the GitLab repo in this comment! https://gitlab.com/fdroid/rfp/-/issues/2732#note_1931850811

I'll also make sure Obtanium works as well in the meantime. You can (or the F-Droid maintainers) can reach out either on GitLab, here, or via email if anything else is needed.

**Comment by haileyok** [comment]
Another update: I've added releases to the GitHub Releases page (https://github.com/bluesky-social/social-app/releases/latest) so you can use Obtanium. Will keep track of responses from the F-Droid team as well.

<img src="https://github.com/bluesky-social/social-app/assets/153161762/0723993d-b23a-4507-837e-1c497f0ad2c4" width="200">

Leaving this issue open for now pending F-Droid status.

**Comment by Mattis142** [comment]
are there any updates..? I mean I'd assume f-droid would've responded by now right?

**Comment by Mattis142** [comment]
yes I get that, I still would've liked to at least hear their reply on the matter yk

**Comment by linsui** [comment]
Why do you ping me here? We are waiting for the answer from the dev. I can do nothing.

**Comment by linsui** [comment]
I draft an MR https://gitlab.com/fdroid/fdroiddata/-/merge_requests/20796. I have to remove many non-free deps. Please help test the apk built by the CI.

**Comment by LeOS-GSI** [comment]
what is the build command for f-droid release without google firebase / play install and all this spying trackers ?

**Comment by LeOS-GSI** [comment]
> Bluesky is made available through [F-Droid](https://f-droid.org/).

It is NOT available on F-droid or Izzy or any other FOSS repo !!!

**Comment by parcelcat** [comment]
For everyone waiting for a Bluesky app on F-Droid, there are now three microblogging clients in the main F-Droid repository that support Bluesky: [Fread](https://github.com/0xZhangKe/Fread), [Flare](https://github.com/DimensionDev/Flare), and [Dhaaga](https://github.com/suvam0451/dhaaga). While I would still like to see Bluesky properly release its official app on F-Droid, it's good to see the community take things into its own hands.

**Comment by linsui** [comment]
Five.  There are also WAFRN and Nora.
